### PR TITLE
Installed react form hook

### PR DESCRIPTION
**Pull request by annamarika**
Added a Form to create a new Object. Please notice you can only use images by unsplash for the image url (next.js thing). It will be replaced by a upload button later on. Feel free to use the following code for testing https://images.unsplash.com/photo-1574201635302-388dd92a4c3f?ixlib=rb-1.2.1&raw_url=true&q=80&fm=jpg&crop=entropy&cs=tinysrgb&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1530

### CODE REVIEW

File: src/components/UI/Form/StyledInputContainer.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+const InputContainer = styled.div`
+	text-transform: uppercase;
+	color: var(--text-color-dark);
+	width: 383px;
```
Comment (anitaschiller) [inline src/components/UI/Form/StyledInputContainer.js:6]: Is it planned that your form has a specific width and height? It looks like your input are cut on the right side but that's maybe part of the design?

**Review by anitaschiller — CHANGES_REQUESTED** [review]
Hi Anna, really nice work! 😊
Just some minor remarks from my side.

File: pages/index.js [hunk 2]
```diff
@@ -11,6 +14,27 @@ export function getStaticProps() {
 	};
 }
 
-export default function Home({ products }) {
-	return <ProductGrid products={products} />;
+export default function Home() {
+	const [products, setProducts] = useState(getProducts);
```
[the remark this answers is not in the document]
Comment (annamarika) [reply pages/index.js:18]: Thanks, I also like it better that way.

File: src/components/UI/Form/StyledInputContainer.js (hunk 1 above)
Comment (annamarika) [reply src/components/UI/Form/StyledInputContainer.js:6]: Hey Anita, this is part of the design. I showed it to the others, I will change it so there is no confusing

**Review by annamarika — COMMENTED** [review]
Hey Anita, I updated your comments, please check ;)

**Review by HishamAlmannai — APPROVED** [review]
Well done!
next Step: Q&A

**Pull request merged**